### PR TITLE
gh actions

### PR DESCRIPTION
**Pull request by jacobzlogar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "export": "next build && next export"
```
Comment (elbojoloco) [inline package.json]: Do we need export if we're not going to use SSG?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```
Comment (elbojoloco) [inline .github/workflows/ci.yml:21]: Actually don't have much experience with GH actions, can't we use a Node docker image here instead?

Comment (jacobzlogar) [reply .github/workflows/ci.yml:21]: me neither; I think we could we'd just have to build and publish the image - i can look into that:

(might be useful) https://snyk.io/blog/managing-node-js-docker-images-in-github-packages-using-github-actions/

File: package.json (hunk 1 above)
Comment (jacobzlogar) [reply package.json]: i don't think we would, i'll remove it

**Review by elbojoloco — APPROVED** [review]

**Comment by elbojoloco** [comment]
Are we good to merge?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (koenhoeijmakers) [reply .github/workflows/ci.yml:21]: No, this defines what virtual machine the tests are ran on, you can choose github's predefined or host your own.

[reference](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on)

**Pull request merged**